utils/answer-types.js

```diff
@@ -1232,7 +1233,6 @@ Khan.answerTypes = $.extend(Khan.answerTypes, {
                         guess: guess
                     };
                 } else {
-                    score.empty = false;
```

Comment:
That was keeping some things from being empty / unanswered.

First, at [line 1196](https://github.com/Khan/khan-exercises/blob/master/utils/answer-types.js#L1196) there is the `checkIfAnswerEmpty` test. In simple cases it will work, but as it checks raw posted values (before letting subproblems to process them) it's not reliable.

Second, if the multiple happens to have a rational-accepting subproblem (such as the default `number` type), that has `required` simplification (that's also the default), then entering an unsimplified correct result makes the score empty-with-message (through the [block-grading-message-branch](https://github.com/Khan/khan-exercises/blob/master/utils/answer-types.js#L1208)).

Third, there is the case of an empty score with an empty-string message, due to an unanswered custom subproblem (that is graded incorrect -- see below).